source/PlayerInfoPanel.h

```diff
@@ -67,7 +65,7 @@ class PlayerInfoPanel : public Panel {
 	// Try to scroll to the given position. Return true if position changed.
 	bool ScrollAbsolute(int scroll);
 
-	void SortShips(InfoPanelState::ShipComparator &shipComparator);
+	void SortShips(InfoPanelState::ShipComparator *shipComparator);
```

Comment:
I think this function needs the shipComparator to function properly, and to me that means having a reference and not a pointer.
Or if we should be able to get a pointer then we need to check it isn't nullptr before doing all we do in that function.

Reply:
It's a pointer cause it needs to be reassignable so I think in this case it's safe to assume that shipComparator is not nullptr. Maybe  can add an assert.